monitor_other_boards.py

```diff
@@ -6,6 +6,7 @@
 
 import monitor_config
 import slack_reporter
+import stack_printing  # Set up the ability to print stack traces on SIGUSR1
```

Comment:
This line is unrelated to last night's troubles. It felt like the right thing to have on general principles, just in case. but I doubt it'll ever be used.